### PR TITLE
fix(ObjectPage): Highlight selected section

### PR DESCRIPTION
**Pull request by MarcusNotheis**
- Highlight the selected Section on scroll
- Add `noHeader` prop

### CODE REVIEW

File: packages/main/package.json [hunk 1]
```diff
@@ -22,6 +22,8 @@
   "dependencies": {
     "@ui5/webcomponents": "1.0.0-rc.2",
     "@ui5/webcomponents-react-base": "0.3.2-rc.14",
+    "lodash.debounce": "^4.0.8",
+    "lodash.throttle": "^4.1.1",
```
Comment (vbersch) [inline packages/main/package.json]: not needed anymore

File: packages/main/src/components/ObjectPage/index.tsx [hunk 2]
```diff
@@ -21,286 +34,232 @@ export interface ObjectPagePropTypes extends CommonProps {
...
-          const { id } = e.getParameter('props');
-          requestAnimationFrame(() => {
-            this.scrollToSectionWithId(id);
-          });
-        }
-      }
-    );
+  useEffect(() => {
+    if (mode === ObjectPageMode.Default) {
+      console.log('forced reflow');
```
Comment (vbersch) [inline packages/main/src/components/ObjectPage/index.tsx]: console statement

File: packages/main/package.json (hunk 1 above)
Comment (MarcusNotheis) [reply packages/main/package.json]: `throttle` is removed, `debounce` is used

File: packages/main/src/components/ObjectPage/index.tsx (hunk 2 above)
Comment (MarcusNotheis) [reply packages/main/src/components/ObjectPage/index.tsx]: removed

**Comment by MarcusNotheis** [comment]
Scrolling to the bottom of the last section is now working in `noHeader` mode as well.

**Pull request merged**